homeassistant/components/romy/strings.json

```diff
@@ -0,0 +1,52 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "unlock_http_interface": {
+        "title": "Local Http Interface is locked",
```

Comment:
This seems a little low level / technical for users. I think we can just make this more about asking the user to enter the password for the device?

Reply:
sure (i am too technical biased for that), changed it now to:
![pw_new](https://github.com/home-assistant/core/assets/2132412/09ea9763-21df-424a-8b70-c5043ac879fe)